esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/compare/bin/compare_task_runner.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Send the output from 'set -x' to 'stdout' rather than 'stderr'.
+BASH_XTRACEFD=1
+#set -eux
+
+FOLDER_NAME=$(find "${OUTPUT_DIR}" -type d -name "recipe_${KGO_METRIC}*")
+
+# Fix this line later
+cd /net/project/ukmo/scitools/opt_scitools/conda/environments/esmvaltool-2.6.0/lib/python3.10/site-packages/esmvaltool
```

Comment:
As we talked about, nervous about a full path here

Reply:
I figured better safe than sorry and went ahead and removed it just in case. I've left a comment to put the path to installation in and saved the path elsewhere for now. 70f77c41a